<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BottomBar</class>
 <widget class="QWidget" name="BottomBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>44</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="signalBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="syncLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>13</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>10</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Currently there is no sync block source.</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lockBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="promptlabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Synchronizing data</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel2_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Wallet Synchronization here</string>
   </property>
  </widget>
  <widget class="QLabel" name="promptlabel_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Synchronizing wallet</string>
   </property>
  </widget>
  <widget class="QLabel" name="syncLabel_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
